lib/haskell/natural4/src/LS/NLP/NL4Transformations.hs

```diff
@@ -220,15 +220,18 @@ mapBSLabel ::
   AA.BoolStruct (Maybe (AA.Label b)) d
 mapBSLabel f g = \case
   AA.Leaf x -> AA.Leaf $ g x
+  AA.Any (Just (AA.Metadata _)) xs -> go AA.Any Nothing xs
+  AA.All (Just (AA.Metadata _)) xs -> go AA.All Nothing xs
```

Comment:
Why are we dropping the metadata here again?

Reply:
The short answer is, because this code is deprecated anyway. 😅

Long answer: This is for a natural language generation, and it was written with the assumption that the label is text that is useful for natural language. So an example would be
```
pre="consumes"
"an alcoholic", "a non-alcoholic"
post="beverage"
```
and it's meant to make those parts into a proper sentence. Metadata, on the other hand, is never something that is written in the spreadsheet, and thus won't be used for NLG.